.travis.yml

```diff
@@ -9,6 +10,10 @@ env:
 - BUILDER_NAME="ovs-cni/${TRAVIS_JOB_ID}builder"
 
 install: true
+script:
+  - cd ../..
+  - mv kubevirt ../kubevirt.io
+  - cd ../kubevirt.io/ovs-cni
 
 deploy:
 - provider: script
```

Comment:
don't you need to `cd` to the project folder? `make docker-build` triggers `make build`.

Reply:
Could you please try it on your environment? This looks error prone.

Reply:
I meant trying it on your repo and your Travis. If it is not a big problem